### PR TITLE
fix: multiple line input and alignment of apply coupons in text box

### LINKED ISSUE #2621: bug: multi line input and alignment of apply coupons
**Describe the bug**
The alignment of the apply coupons Edit text is not proper also there is multi line input for it.

**To Reproduce**
1. Go to Tickets
2. Click on apply coupons
3. Input multi line text

**Expected behavior**
The text must be single line as all promo codes or discount coupons are single line text. Also the editText and button must be aligned properly

**Screenshots**
Screenshot_2020-02-01-12-33-06-525_com eventyay attendee

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Redmi Note 4|
|Android Version|Nougat 7.0 |

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by Dhruv-194**
Fixes #2621 

Multi line input and alignment of apply coupons.

**The bug was -** 
The alignment of the apply coupons Edit text is not proper also there is multi line input for it.

**Now after fixing -** 
I fixed the 'Apply Discount Coupon' EditText in "fragment_tickets.xml" file where now only single line input will be allowed. Further I aligned the EditText with 'Apply' Button.

Changes: The changes were put into the "fragment_tickets.xml" file.

 **Screenshot-**
Screenshot_20200510-031234

### CODE REVIEW

**Comment by iamareebjamal** [comment]
No need to list changes in PR, we can see them in proper diff. Please add screenshots of the change instead. Also, your PR title is not following project conventions. Please see titles of merged PRs

**Comment by iamareebjamal** [comment]
Also, your PR doesn't link the fixed issue correctly. Please see merged PRs

**Comment by Dhruv-194** [comment]
> No need to list changes in PR, we can see them in proper diff. Please add screenshots of the change instead. Also, your PR title is not following project conventions. Please see titles of merged PRs

I have changed the title but still it is not considering it. Moreover as I am new and this is my first contribution ever, can you please tell me how to link this PR properly? I have looked into many merged PR but I am unable to find out what I am missing.

**Comment by iamareebjamal** [comment]
Look what's the pattern of titles in merged PRs and how the issue is linked. You need to look **exactly**. See other open PRs how they are writing the title for bug fixes and linking issues

**Comment by iamareebjamal** [comment]
Literally any open bug fix will do

https://github.com/fossasia/open-event-attendee-android/pull/2636
https://github.com/fossasia/open-event-attendee-android/pull/2637
https://github.com/fossasia/open-event-attendee-android/pull/2349
https://github.com/fossasia/open-event-attendee-android/pull/2617
https://github.com/fossasia/open-event-attendee-android/pull/2587

**Comment by Dhruv-194** [comment]
> Literally any open bug fix will do
> 
> #2636
> #2637
> #2349
> #2617
> #2587

I tried merging the entire thing again with correct semantic of commit message and the PR title. 
But still it is not considering. I don't know what to do now
If you know anything please tell otherwise I won't be able to contribute more than this.

**Comment by iamareebjamal** [comment]
No merging of anything is required. You have to change the PR title to follow the pattern and link the issue in the PR message in the standard way github provides. I have already linked multiple PRs of new contributors who have done it as well. And you'll have to link issues in this way on **any** GitHub project, so doesn't matter if you learn it today or tomorrow, Open Source is not just about sending PRs, it's also about following project conventions and learning things by looking around

Neither your PR title follows the convention, nor you have linked the issue in the PR body like in other PRs. You have to do it **exactly** the way others have done

You are a programmer, and you should already understand the strict nature of languages and context-sensitive information. If the command to print a line is `System.out.println("messages")`, you can't write `Print.the line (message), pretty please`. So please, follow the conventions like other people have and link the issue **exactly** as others have, it has to be **word to word**, you can't add your choice of words in between just for fun, it has to be **exact**. And you have to follow the PR title convention, like other people have. If you can't see the difference between `
Fixed issue: multi line input and alignment of apply coupons` and `fix: added a scroll view in past ticket filter`, I am sorry, I can't help you further. 

I can change it myself in 2 seconds or tell you exactly what to do, but then you won't learn how to join and follow the conventions of any new community you contribute to. These are the things you'll eventually have to learn about any community

**Comment by Dhruv-194** [comment]
> No merging of anything is required. You have to change the PR title to follow the pattern and link the issue in the PR message in the standard way github provides. I have already linked multiple PRs of new contributors who have done it as well. And you'll have to link issues in this way on **any** GitHub project, so doesn't matter if you learn it today or tomorrow, Open Source is not just about sending PRs, it's also about following project conventions and learning things by looking around
> 
> Neither your PR title follows the convention, nor you have linked the issue in the PR body like in other PRs. You have to do it **exactly** the way others have done
> 
> You are a programmer, and you should already understand the strict nature of languages and context-sensitive information. If the command to print a line is `System.out.println("messages")`, you can't write `Print.the line (message), pretty please`. So please, follow the conventions like other people have and link the issue **exactly** as others have, it has to be **word to word**, you can't add your choice of words in between just for fun, it has to be **exact**. And you have to follow the PR title convention, like other people have. If you can't see the difference between ` Fixed issue: multi line input and alignment of apply coupons` and `fix: added a scroll view in past ticket filter`, I am sorry, I can't help you further.
> 
> I can change it myself in 2 seconds or tell you exactly what to do, but then you won't learn how to join and follow the conventions of any new community you contribute to. These are the things you'll eventually have to learn about any community

I have made the necessary changes by following the convention. Kindly please review it. Thanks!

**Comment by Dhruv-194** [comment]
And also one more thing, the first commit was the right one and the last one has not changes.

**Comment by iamareebjamal** [comment]
There are no changes in the PR https://github.com/fossasia/open-event-attendee-android/pull/2698/files

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/fossasia/open-event-attendee-android/pull/2698">fix: multiple line input and alignment of apply coupons in text box by Dhruv-194 · Pull Request #2698 · fossasia/open-event-attendee-android</a></strong></div><div>Fixes #2621
Multi line input and alignment of apply coupons.
The bug was -
The alignment of the apply coupons Edit text is not proper also there is multi line input for it.
Now after fixing -
I fix...</div></blockquote>

**Comment by Dhruv-194** [comment]
Actually the first commit which I made had all the necessary changes after that I commited by mistake becuase I thought there was something wrong with my commit message. The last commit does not have any changes. So is there any way to revert back to the first commit where I have made all the changes?

**Comment by iamareebjamal** [comment]
https://git-scm.com/docs/git-reset

<blockquote><div><strong><a href="https://git-scm.com/docs/git-reset">Git - git-reset Documentation</a></strong></div></blockquote>

**Comment by Dhruv-194** [comment]
> There are no changes in the PR https://github.com/fossasia/open-event-attendee-android/pull/2698/files
> 
> >  GitHub**[fix: multiple line input and alignment of apply coupons in text box by Dhruv-194 · Pull Request #2698 · fossasia/open-event-attendee-android](https://github.com/fossasia/open-event-attendee-android/pull/2698)**Fixes #2621
> > Multi line input and alignment of apply coupons.
> > The bug was -
> > The alignment of the apply coupons Edit text is not proper also there is multi line input for it.
> > Now after fixing -
> > I fix...

Sir Can you please check now

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/fossasia/open-event-attendee-android/pull/2698">fix: multiple line input and alignment of apply coupons in text box by Dhruv-194 · Pull Request #2698 · fossasia/open-event-attendee-android</a></strong></div><div>Fixes #2621
Multi line input and alignment of apply coupons.
The bug was -
The alignment of the apply coupons Edit text is not proper also there is multi line input for it.
Now after fixing -
I fix...</div></blockquote>

File: app/src/main/res/layout/fragment_tickets.xml [hunk 1]
```diff
@@ -137,12 +137,20 @@
                     android:layout_width="0dp"
                     android:layout_weight="1"
                     android:layout_height="wrap_content"
+
+                    android:inputType="text"
+                    android:maxLines="1"
+                    android:layout_gravity="left"
+
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_tickets.xml]: Remove empty lines

File: app/src/main/res/layout/fragment_tickets.xml [hunk 2]
```diff
@@ -137,12 +137,20 @@
...
                     android:layout_marginStart="@dimen/layout_margin_medium"/>
 
                 <com.google.android.material.button.MaterialButton
                     android:id="@+id/applyButton"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+
+                    android:layout_gravity="right"
+
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_tickets.xml]: Here as well

**Comment by iamareebjamal** [comment]
> No need to list changes in PR, we can see them in proper diff. Please add screenshots of the change instead.

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-attendee-android/pull/2698#issuecomment-626219690

**Comment by Dhruv-194** [comment]
> [#2698 (comment)](https://github.com/fossasia/open-event-attendee-android/pull/2698#issuecomment-626219690)

Done sir. Kindly review it now.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**